environment-dev.yml

```diff
@@ -1,5 +1,6 @@
 name: pylops
 channels:
+  - https://software.repos.intel.com/python/conda
```

Comment:
Same as above, would this also install intel `numpy` over the one from coda-forge? 

If so, I feel this may be too strong of a change as now `Intel` takes priority over anything and we should maybe provide an environment specific for Intel?